stdlib/3/asyncio/events.pyi

```diff
@@ -327,8 +331,9 @@ def get_event_loop() -> AbstractEventLoop: ...
 def set_event_loop(loop: Optional[AbstractEventLoop]) -> None: ...
 def new_event_loop() -> AbstractEventLoop: ...
 
-def get_child_watcher() -> Any: ...  # TODO: unix_events.AbstractChildWatcher
-def set_child_watcher(watcher: Any) -> None: ...  # TODO: unix_events.AbstractChildWatcher
+if sys.platform != 'win32':
```

Comment:
Making this Windows-only seems like an OK solution, but there should be a comment explaining why we do it that way, even though this function does exist at runtime in Windows.